xep-0373.xml

```diff
@@ -337,6 +345,11 @@
     public key using Base64. The id of the node MUST be "urn:xmpp:openpgp:0:public-keys:" followed
     by the fingerprint string of the OpenPGP public-key contained in the data node.</p>
 
+    <p>It is REQUIRED to set the access model of the public key data node to ‘open’ to give
```

Comment:
REQUIRED feels to strong here. There may be valid use cases where a different access model is sensible, e.g. a whitelist based one. I think I would go with with RECOMMENDED, maybe even like "In absence of a use-case specific access model, it is RECOMMENDED to use the 'open' access model, to give…".